prov/efa/src/rxr/rxr_pkt_cmd.c

```diff
@@ -680,7 +695,10 @@ void rxr_pkt_handle_send_completion(struct rxr_ep *ep, struct fi_cq_data_entry *
 	case RXR_DC_EAGER_TAGRTM_PKT:
 	case RXR_DC_MEDIUM_MSGRTM_PKT:
 	case RXR_DC_MEDIUM_TAGRTM_PKT:
+	case RXR_DC_LONG_MSGRTM_PKT:
+	case RXR_DC_LONG_TAGRTM_PKT:
 	case RXR_DC_EAGER_RTW_PKT:
+	case RXR_DC_LONG_RTW_PKT:
```

Comment:
I think for ```DC_LONG``` packet, you need to implement the send_completion, and increase ```tx_entry->bytes_acked``` there, otherwise in ```rxr_pkt_hand_data_send_completion```,  ```tx_entry->total_len``` will not be same as ```tx_entry->bytes_acked```.

Reply:
Send completions for long rtm packet and long rtw packet has been introduced. Resolving.